specs/image-update-monitoring.md

```diff
@@ -0,0 +1,32 @@
+# Docker Image Update Monitoring
+
+## Summary
+
+Many GitOps users would like to automate Kubernetes manifest changes in the deployment repository
+(see [Deployment Repo Update Automation](./deployment-repo-update.md)). The changes might be triggered by
+the CI pipeline run or a new image in the Docker registry. The [Flux] provides the docker registry monitoring as part of
```

Comment:
s/The [Flux]/Flux/

s/the docker registry monitoring/docker registry monitoring/

Reply:
fixed